### PR TITLE
[FIX] Fixed the Syntax Error, code : QUERY_SYNTAX_ERROR

### PR DESCRIPTION
**Pull request by saif-serpentcs**
**Error** : {u'success': False, u'error': {u'message': u"Syntax Error on line 1: token '-02' Unexpected COLUMNNAME(-02), expected one of: SEMICOLON", u'code': u'QUERY_SYNTAX_ERROR'}}

**Fixed** in **V10**.

@JayVora-SerpentCS Please **review** and **merge**?

### CODE REVIEW

File: vtiger_connector_crm/models/res_company.py [hunk 1]
```diff
@@ -21,8 +21,7 @@ def sync_vtiger_crm(self):
             company.sync_vtiger_partner()
             access_key = company.get_vtiger_access_key()
             session_name = company.vtiger_login(access_key)
-            qry = """SELECT * FROM Potentials WHERE modifiedtime >= %s;"""\
-                % (company.last_sync_date)
+            qry = """SELECT * FROM Potentials WHERE modifiedtime >= '%s';""" % (company.last_sync_date)
```
Comment (JayVora-SerpentCS) [inline vtiger_connector_crm/models/res_company.py]: pep-8 plz

File: vtiger_connector_partner/models/res_company.py [hunk 2]
```diff
@@ -18,9 +18,9 @@ def action_sync_vtiger(self):
     def sync_vtiger_partner(self):
         for company in self:
             access_key = company.get_vtiger_access_key()
+            print "access_key-----10---------", access_key
```
Comment (JayVora-SerpentCS) [inline vtiger_connector_partner/models/res_company.py]: be serious. remove print

**Comment by JayVora-SerpentCS** [comment]
Did you fix the pep-8?

**Comment by saif-serpentcs** [comment]
Nope sir, I will complete it ASAP

**Comment by saif-serpentcs** [comment]
Fixed pep8 v10

**Comment by saif-serpentcs** [comment]
@JayVora-SerpentCS 
Could you please review and merge it v10?
Thanks.

**Pull request merged**